functions/.eslintrc.cjs

```diff
@@ -21,6 +21,7 @@ module.exports = {
     'prettier',
   ],
   rules: {
+    'no-unused-vars': 'off',
```

Comment:
Would be great if we can identify that we use this rule but find a way that it doesn't flag the interface definitions by mistake. If that is not possible we should rather add exceptions just for these files.

Reply:
There are two different rules here, one is 'no-unused-vars' and one is '@typescript-eslint/no-unused-vars', this disabling only disables the first one which is JS-specific, but not the TS-specific one - since this change may not be required after all though, I will try to remove it all together